## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 516377`
Commit: `785969744e16db18a6f49ab2572ee10b900a842c`

---
### `x86_64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>fence</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>fence</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>fence</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>fence</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>fence</summary>
<pre>--- FAIL: TestIntegration_EnvWorks (0.14s)
    integration_test.go:539: sandbox initialization failed: exit status 1
    integration_test.go:539: expected command to succeed, but it failed with exit code 1
        stdout: 
        stderr: bwrap: Unable to set up system call filtering as requested: prctl(PR_SET_SECCOMP) reported EINVAL. (Hint: this requires a kernel configbwrap: ured with CONFIG_SECCOMP and CONFIG_SECCOMP_FILTER.)
        Unable to set up system call filtering as requested: prctl(PR_SET_SECCOMP) reported EINVAL. (Hint: this requires a kernel configured with CONFIG_SECCOMP and CONFIG_SECCOMP_FILTER.)
        
        error: exit status 1
    integration_test.go:540: expected &quot;&quot; to contain &quot;FENCE_SANDBOX=1&quot;
[fence:linux] runtime exec deny warning for /nix/store/jjxngswsb214vb58qx485jhmilf0kxxy-coreutils-9.10/bin/coreutils (requested: chroot): shared binary also implements critical commands [cat head tail +30 more detected aliases, use --debug for expanded details], which will be collaterally blocked. To skip runtime blocking of &quot;chroot&quot; and silence this warning, add it to &quot;acceptSharedBinaryCannotRuntimeDeny&quot; in your command config.
[fence:linux] runtime exec deny warning for /nix/store/jjxngswsb214vb58qx485jhmilf0kxxy-coreutils-9.10/bin/coreutils (requested: chroot): shared binary also implements critical commands [cat head tail +30 more detected aliases, use --debug for expanded details], which will be collaterally blocked. To skip runtime blocking of &quot;chroot&quot; and silence this warning, add it to &quot;acceptSharedBinaryCannotRuntimeDeny&quot; in your command config.
[fence:linux] runtime exec deny warning for /nix/store/jjxngswsb214vb58qx485jhmilf0kxxy-coreutils-9.10/bin/coreutils (requested: chroot): shared binary also implements critical commands [cat head tail +30 more detected aliases, use --debug for expanded details], which will be collaterally blocked. To skip runtime blocking of &quot;chroot&quot; and silence this warning, add it to &quot;acceptSharedBinaryCannotRuntimeDeny&quot; in your command config.
[fence:linux] runtime exec deny warning for /nix/store/jjxngswsb214vb58qx485jhmilf0kxxy-coreutils-9.10/bin/coreutils (requested: chroot): shared binary also implements critical commands [cat head tail +30 more detected aliases, use --debug for expanded details], which will be collaterally blocked. To skip runtime blocking of &quot;chroot&quot; and silence this warning, add it to &quot;acceptSharedBinaryCannotRuntimeDeny&quot; in your command config.
[fence:linux] runtime exec deny warning for /nix/store/jjxngswsb214vb58qx485jhmilf0kxxy-coreutils-9.10/bin/coreutils (requested: chroot): shared binary also implements critical commands [cat head tail +30 more detected aliases, use --debug for expanded details], which will be collaterally blocked. To skip runtime blocking of &quot;chroot&quot; and silence this warning, add it to &quot;acceptSharedBinaryCannotRuntimeDeny&quot; in your command config.
[fence:linux] runtime exec deny warning for /nix/store/jjxngswsb214vb58qx485jhmilf0kxxy-coreutils-9.10/bin/coreutils (requested: chroot): shared binary also implements critical commands [cat head tail +30 more detected aliases, use --debug for expanded details], which will be collaterally blocked. To skip runtime blocking of &quot;chroot&quot; and silence this warning, add it to &quot;acceptSharedBinaryCannotRuntimeDeny&quot; in your command config.
[fence:linux] runtime exec deny warning for /nix/store/jjxngswsb214vb58qx485jhmilf0kxxy-coreutils-9.10/bin/coreutils (requested: chroot): shared binary also implements critical commands [cat head tail +30 more detected aliases, use --debug for expanded details], which will be collaterally blocked. To skip runtime blocking of &quot;chroot&quot; and silence this warning, add it to &quot;acceptSharedBinaryCannotRuntimeDeny&quot; in your command config.
[fence:linux] runtime exec deny warning for /nix/store/jjxngswsb214vb58qx485jhmilf0kxxy-coreutils-9.10/bin/coreutils (requested: chroot): shared binary also implements critical commands [cat head tail +30 more detected aliases, use --debug for expanded details], which will be collaterally blocked. To skip runtime blocking of &quot;chroot&quot; and silence this warning, add it to &quot;acceptSharedBinaryCannotRuntimeDeny&quot; in your command config.
FAIL
FAIL	github.com/Use-Tusk/fence/internal/sandbox	4.910s
FAIL</pre>
</details>
</details>

--- a/pkgs/by-name/fe/fence/package.nix
+++ b/pkgs/by-name/fe/fence/package.nix
@@ -25,6 +25,10 @@ buildGoModule (finalAttrs: {
 
   vendorHash = "sha256-Qct/M0zuggYzlN0gyO8nF58M5Av3HcYyMjrPab5Crr0=";
 
+  subPackages = [
+    "cmd/fence"
+  ];
+
   ldflags = [
     "-s"
     "-w"

